paddle2onnx/graph/paddle_graph.py

```diff
@@ -120,6 +120,13 @@ def make_node(self,
         if attrs is None:
             attrs = kw
         attrs.update(kw)
+        if op.type == 'conv2d':
+            if attrs['data_format'] == 'NCHW':
+                pass
+            else:
+                raise TypeError(
+                    "The input data format should be 'NCHW', but received data format is %s."
+                    % attrs['data_format'])
```

Comment:
Let's put this check at `op_mapper/nn.py::conv2d`, just where we start to convert `conv2d` operator, and use `assert` to do this.

Reply:
has fixed